drivers/storage/portworx/util/csi_generator.go

```diff
@@ -195,6 +200,11 @@ func (g *CSIGenerator) GetCSIConfiguration() *CSIConfiguration {
 		cv.IncludeSnapshotController = false
 	}
 
+	// IncludeExternalHealthMonitor only with PX 2.8.0+ and k8s 1.21+
+	if g.kubeVersion.GreaterThanOrEqual(k8sVer1_21) && g.pxVersion.GreaterThanOrEqual(pxVer2_8) {
```

Comment:
If we are going to support this from px 2.8.0, remember to add the health monitor controller image to old spec generators (/version endpoint) from 2.8.0 and above.

Reply:
Switched this to be px 2.10+